vignettes/custom_appearance.Rmd

```diff
@@ -381,4 +382,63 @@ basic_table(
-```
\ No newline at end of file
+```
+
+### Cell, Label, and Annotation Wrapping
+
+An `rtable` can be rendered with a customized width by setting custom rendering widths for cell contents, row labels, and titles/footers.
+
+This is demonstrated using the sample data and table below. In this section we aim to render this table with a reduced width since the table has very wide contents in several cells, labels, and titles/footnotes.
+
+```{r}
+trimmed_data <- ex_adsl %>%
+    filter(SEX %in% c("M", "F")) %>%
+    filter(RACE %in% levels(RACE)[1:2])
+
+levels(trimmed_data$ARM)[1] <- "Incredibly long column name to be wrapped"
+levels(trimmed_data$ARM)[2] <- "This_column_name_should_be_split_somewhere"
+
+wide_tbl <- basic_table(
+    title = "Title that is too long and also needs to be wrapped to a smaller width",
+    subtitles = "Subtitle that is also long and also needs to be wrapped to a smaller width",
+    main_footer = "Footnote that is wider than expected for this table.",
+    prov_footer = "Provenance footer material that is also wider than expected for this table.") %>%
+    split_cols_by("ARM") %>%
+    split_rows_by("RACE", split_fun = drop_split_levels) %>%
+    analyze(c("AGE", "EOSDY"),
+            na_str = "Very long cell contents to_be_wrapped_and_splitted",
+            inclNAs = TRUE) %>%
+    build_table(trimmed_data)
+
+wide_tbl
+```
+
+In the following sections we will use the `formatters::matrix_form()` function to transform the `rtable` into matrix form (preserving any indentation) and then the `toString()` function to render this matrix in string form. This resulting string representation is ready to be printed or written to a plain text file, but we will use the `strsplit()` function in combination with the `matrix()` function to preview the rendered wrapped table in matrix form within this vignette.
+
+#### Cell & Label Wrapping
+
+The width of a rendered table can be customized by wrapping column widths. This is done by setting custom width values via the `widths` argument of the `toString()` function. The length of the vector passed to the `widths` argument must be equal to the total number of columns in the table, including the row labels column, with each value of the vector corresponding to the maximum width (in characters) allowed in each column, from left to right.
+
+For example, `wide_tbl` has four columns (1 row label column and 3 content columns) which we will set the widths of below to use in the rendered table. We set the width of the row label column to 10 characters and the widths of each of the 3 content columns to 8 characters. Any words longer than the specified width are broken and continued on the following line. By default there are 3 spaces separating each of the columns in the rendered table but this can be customized via the `col_gap` argument to `toString()` if further width customization is desired.
+
+```{r}
+result_wrap_cells <- toString(matrix_form(wide_tbl, indent_rownames = TRUE),
```

Comment:
we shouldn't need to manually create the matrix form here, calling `toString` with the `widths` set should be sufficient. End users shouldn't need to worry about the matrix form, thats more for developers.

Reply:
Updated.